linkerd/proxy/http/src/detect.rs

```diff
@@ -136,8 +137,10 @@ where
             Some(HttpVersion::Http1) => {
                 debug!("Handling as HTTP");
                 let http1 = if let Some(svc) = self.http1.clone() {
+                    trace!("HTTP service already exists");
```

Comment:
these were also from debugging but i thought they might actually be worth having.